boards/arm/mimx8mm_evk_a53/doc/index.rst

```diff
@@ -0,0 +1,111 @@
+.. _imx8mm_evk_a53:
+
+NXP i.MX8MM EVK (Cortex-A53)
+#################################
+
+Overview
+********
+
+i.MX8M Mini LPDDR4 EVK board is based on NXP i.MX8M Mini applications
+processor, composed of a quad Cortex®-A53 cluster and a single Cortex®-M4 core.
+Zephyr OS is ported to run on the Cortex®-M4 core.
```

Comment:
Did you mean to say that Zephyr already runs on the M4, but now it can also run on the A53? It's a bit confusing as it is written now.

Reply:
It runs on A53 now.